pennylane/qnn/keras.py

```diff
@@ -0,0 +1,292 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module is for PennyLane-Keras integration."""
+import functools
+import inspect
+from collections.abc import Iterable
+from typing import Optional
+
+try:
+    import tensorflow as tf
+    from tensorflow.keras.layers import Layer
+    from pennylane.interfaces.tf import to_tf
+
+    CORRECT_TF_VERSION = int(tf.__version__.split(".")[0]) > 1
+except ImportError:
+    from abc import ABC
+
+    Layer = ABC
+    CORRECT_TF_VERSION = False
+
+from pennylane.qnodes import QNode
+
+
+class KerasLayer(Layer):
+    """A Keras Layer_ for integrating PennyLane QNodes with the Keras API.
```

Comment:
Looks like this link isn't resolving 🤔 Is the `_` a typo?

Reply:
Nitpick: There could be room for making the first line of the docstring better communicate what this class does for users. In fact, your sentence down on line 51 does this perfectly!
```suggestion
    """This class converts a :func:`~.QNode` to a Keras Layer_, which can be used within the Keras
    `Sequential <https://www.tensorflow.org/api_docs/python/tf/keras/Sequential>`__ or
    `Model <https://www.tensorflow.org/api_docs/python/tf/keras/Model>`__ classes for
    creating quantum and hybrid models.
```